schema/handler.go

```diff
@@ -77,34 +81,140 @@ type getJsonSchema struct {
 //       200: jsonSchema
 //       404: jsonError
 //       500: jsonError
-func (h *Handler) get(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+func (h *Handler) getByID(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	s, err := h.r.IdentityTraitsSchemas(r.Context()).GetByID(ps.ByName("id"))
 	if err != nil {
 		h.r.Writer().WriteError(w, r, errors.WithStack(herodot.ErrNotFound.WithDebugf("%+v", err)))
 		return
 	}
-	var src io.ReadCloser
 
-	if s.URL.Scheme == "file" {
-		src, err = os.Open(s.URL.Host + s.URL.Path)
+	src, err := readSchema(s)
+	if err != nil {
+		h.r.Writer().WriteError(w, r, errors.WithStack(herodot.ErrInternalServerError.WithReasonf("The file for this JSON Schema ID could not be found or opened. This is a configuration issue.").WithDebugf("%+v", err)))
+		return
+	}
+	defer src.Close()
+
+	w.Header().Add("Content-Type", "application/json")
+	if _, err := io.Copy(w, src); err != nil {
+		h.r.Writer().WriteError(w, r, errors.WithStack(herodot.ErrInternalServerError.WithReasonf("The file for this JSON Schema ID could not be found or opened. This is a configuration issue.").WithDebugf("%+v", err)))
+		return
+	}
+}
+
+// Raw JSON Schema list
+//
+// swagger:model jsonSchemas
+// nolint:deadcode,unused
+type jsonSchemas []struct {
+	ID     string          `json:"id"`
+	Schema json.RawMessage `json:"schema"`
+}
```

Comment:
Can you define the struct from this type as it's own type? You copy the type definition multiple times, so it would be easier to make this it's own type.
```suggestion
type JSONSchemas []jsonSchema

type jsonSchema struct {
	ID     string          `json:"id"`
	Schema json.RawMessage `json:"schema"`
}
```

Reply:
The problem I see is that `jsonSchema` is already defined as `json.RawMessage`. Changing this would be a breaking change, but I think it is better then.

Reply:
Sounds good